client/a8c-for-agencies/components/sidebar-menu/hooks/use-sites-menu-items.ts

```diff
@@ -39,13 +40,25 @@ const useSitesMenuItems = ( path: string ) => {
 					icon: warning,
 					path: A4A_SITES_LINK,
 					link: A4A_SITES_LINK_NEEDS_ATTENTION,
-					title: translate( 'Needs Attention' ),
+					title: translate( 'Needs attention' ),
 					trackEventProps: {
 						menu_item: 'Automattic for Agencies / Sites / Needs Attention',
 					},
 				},
 				path
 			),
+			createItem(
```

Comment:
I believe we only show this option if we have pending WPCOM sites.

Reply:
You can use this API to fetch the pending list.

`GET /agency/<agency_id>/sites/pending`